modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BaseBatchEngineTaskExecutorTest.java

```diff
@@ -169,27 +168,16 @@ public void createItem(
 
 			LocalDateTime localDateTime = LocalDateTimeUtil.toLocalDateTime(
 				blogPosting.getDatePublished());
-			Optional<Image> imageOptional = Optional.ofNullable(
-				blogPosting.getImage());
+
+			Image image = blogPosting.getImage();
 
 			blogsEntryService.addEntry(
 				blogPosting.getHeadline(), blogPosting.getAlternativeHeadline(),
 				blogPosting.getFriendlyUrlPath(), blogPosting.getDescription(),
 				blogPosting.getArticleBody(), localDateTime.getMonthValue() - 1,
 				localDateTime.getDayOfMonth(), localDateTime.getYear(),
 				localDateTime.getHour(), localDateTime.getMinute(), true, true,
-				new String[0],
```

Comment:
hi @shuyangzhou 
this is taken from the original source. Should I ping code owners to update code according to your changes?

Best Regards